.github/CODEOWNERS

```diff
@@ -114,7 +114,7 @@
 /pkg/services/navtree/ @grafana/backend-platform
 /pkg/services/notifications/ @grafana/backend-platform
 /pkg/services/org/ @grafana/backend-platform
-/pkg/services/playlist/ @grafana/backend-platform
+/pkg/services/playlist/ @grafana/grafana-app-platform-squad
```

Comment:
does it make sense to just move this service to our team?

Reply:
In the context of this PR, yeah, good thinking.